From b0d826f28ae47d22229949e754709e68afe5d83d Mon Sep 17 00:00:00 2001
From: raiph <raiph.mellor@gmail.com>
Date: Thu, 2 Jul 2020 17:30:07 +0100
Subject: [PATCH] Fix 404 and text in New Unicode properties section
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

See https://github.com/Perl/perl5/issues/17881

I found a 404, and an "old" link. I investigated.

My conclusion was UC have landed new TR18 and TR39 since text in section
  New Unicode properties Identifier_Status and Identifier_Type supported
was written.

I've guessed at a suitable update.

Petr Písař: Ported from e02f7c069a8e7dd98b0ec010e9b3c6619b46baf3
upstream commmit.

Signed-off-by: Petr Písař <ppisar@redhat.com>
---
 pod/perldelta.pod | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/pod/perldelta.pod b/pod/perldelta.pod
index b92ea53..bb3d1ef 100644
--- a/pod/perldelta.pod
+++ b/pod/perldelta.pod
@@ -48,12 +48,12 @@ L<perlop/Operator Precedence and Associativity>.
 
 =head2 New Unicode properties C<Identifier_Status> and C<Identifier_Type> supported
 
-Unicode is in the process of revising its regular expression
-requirements: L<https://www.unicode.org/draft/reports/tr18/tr18.html>.
+Unicode has revised its regular expression requirements:
+L<https://www.unicode.org/reports/tr18/tr18-21.html>.
 As part of that they are wanting more properties to be exposed, ones
 that aren't part of the strict UCD (Unicode character database). These
 two are used for examining inputs for security purposes. Details on
-their usage is at L<https://www.unicode.org/reports/tr39/proposed.html>.
+their usage is at L<https://www.unicode.org/reports/tr39/>.
 
 =head2 It is now possible to write C<qr/\p{Name=...}/>, or
 C<qr!\p{na=/(SMILING|GRINNING) FACE/}!>
-- 
2.25.4

